cogeo_mosaic/backends/dynamodb.py

```diff
@@ -136,21 +175,20 @@ def update(
 
         self._update_metadata()
 
-        return
-
-    def _create_table(
-        self, overwrite: bool = False, billing_mode: str = "PAY_PER_REQUEST"
-    ):
-        if self._table_exist():
-            if not overwrite:
-                raise MosaicExists("Table already exist, use `overwrite=True`.")
-            self.table.delete()
-            self.table.wait_until_not_exists()
+    def _create_table(self, billing_mode: str = "PAY_PER_REQUEST", **kwargs: Any):
+        """Create DynamoDB Table."""
```

Comment:
Probably good to document that `**kwargs` get passed on to `self.client.create_table`